### ISSUE TITLE
Comments about bugs on Alpertron calculator 

### ISSUE THREAD

**Issue by alpertron** [RichardAth/Projects#2]
Richard,

First of all I want to thank you for your work on converting my factoring calculator to a console application.

I've read the notes about the program, and it says that the partition function from my calculator is buggy and you replaced it with a different piece of code.

I've just checked that the values of P(10000), P(20000), ... up to p(100000) match the output of PARI-GP calculator.

I would like to know the precise argument that shows the failure, so I can fix my calculator. Of course if you know of other bugs, please list them here or open issues in my GitHub project: https://github.com/alpertron/calculators/issues

Best regards,

Dario Alpern

**Comment by RichardAth** [comment]
Thanks for your comments. 

 

At the time I replaced the partition function there was a bug but you have since fixed it (see attached email). As I already had an adequate partition function (written to solve a project euler problem) I just used that instead of trying to figure out the cause of the problem. I have never bothered to restore your version because I consider the partition function not very important and the version I used is OK.

 

Regards,

 

Richard Atherton
